sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -109,21 +118,23 @@ export class AssessmentControllerService {
    * @param options for NOA.
    * - `includeDocumentNumber` when true document number is mapped
    * to disbursement dynamic data.
+   * - `includeDateSent `  when true date sent is mapped
```

Comment:
Please remove the extra space after includeDateSent

Reply:
Removed